### ISSUE TITLE
displayCards()

### ISSUE THREAD

**Issue by yisheng90** [wdi-sg/wdi-project-1-jerel-lim#1]
Is the for loop  necessary?

**Comment by jerel-lim** [comment]
you are right... just noticed that i don't need it if i just assign the card value to the img src, instead of finding the value and matching it.
